manage_breast_screening/clinics/jinja2/index.html

```diff
@@ -1,35 +1,18 @@
-{% extends 'layout-app.html' %}
-
-{% from 'tag/macro.jinja' import tag %}
-{% from 'components/count/macro.jinja' import appCount %}
-{% from 'components/secondary-navigation/macro.jinja' import app_secondary_navigation %}
-
+{% extends 'layout-app.html' %} {% from 'tag/macro.jinja' import tag %} {% from
+'components/count/macro.jinja' import appCount %} {% from
+'components/secondary-navigation/macro.jinja' import app_secondary_navigation %}
 {% block content %}
 <h1>{{ presenter.heading }}</h1>
 
-{% set ns = namespace() %}
-{% set ns.secondaryNavItems = [] %}
-
-{% for item in [
-  { "id": 'today', "label": 'Today' },
-  { "id": 'upcoming', "label": 'Upcoming' },
-  { "id": 'completed', "label": 'Completed' },
-  { "id": 'all', "label": 'All' }
-] %}
-{% set href %}/clinics/{{ item.id }}{% endset %}
-{% set ns.secondaryNavItems = ns.secondaryNavItems + [{
-  "text": (item.label + " " + appCount(presenter.counts_by_filter[item.id])) | safe,
-  "href": href | trim,
-  "current": true if item.id == presenter.filter
-}] %}
-{% endfor %}
-
-{{ app_secondary_navigation({
-  "visuallyHiddenTitle": "Secondary menu",
-  "items": ns.secondaryNavItems
-}) }}
-
-{% if presenter.clinics | length == 0 %}
+{% set ns = namespace() %} {% set ns.secondaryNavItems = [] %} {% for item in [
```

Comment:
This is impossible to read. Can you try renaming this file to `.jinja`? That should exclude it from prettier and give you proper syntax highlighting in VSCode.

If we format these files it should be with something that understands Jinja, like [djlint](https://www.djlint.com/)